main.go

```diff
@@ -149,6 +159,9 @@ func visit(url *url.URL) {
 	if err != nil {
 		log.Fatalf("unable to create request: %v", err)
 	}
+	for _, h := range httpHeaders {
+		req.Header.Add(headerKeyValue(h))
```

Comment:
Hi, I came from https://github.com/davecheney/httpstat/pull/33 . Let's merge both features.  Can you handle `Host` header here ?  `Host` header can not be set via `req.Header.Add`. It needs to set via `req.Host`...